includes/core/classes/class-setup.php

```diff
@@ -679,4 +680,29 @@ public function set_event_archive_labels( array $post_states, WP_Post $post ): a
 		return $post_states;
 	}
 
+	/**
+	 * Display a notification to recommend enabling user registration for GatherPress functionality.
+	 *
+	 * This method checks if user registration is enabled in WordPress settings and displays a
+	 * notification encouraging users to enable registration for optimal GatherPress functionality.
+	 * Users have the option to suppress this notification permanently.
+	 *
```

Comment:
Sorry, add since too.

```
/**
 * Display a notification to recommend enabling user registration for GatherPress functionality.
 *
 * This method checks if user registration is enabled in WordPress settings and displays a notification
 * encouraging users to enable registration for optimal GatherPress functionality. Users have the option
 * to suppress this notification permanently.
 *
 * @since 1.0.0
 *
 * @return void
 */
```